site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -167,7 +167,7 @@ Rules are triggered by requests sent from web pages. If multiple rules match a p
 Thinking of matching this way: whatever rule a particular extension prioritizes will then be prioritized against rules from other extensions.
 
 {% Aside 'caution' %}
-Avoid building rulesets that depend on rules with the same priority running in a particular order. Rules with the same priority in the same set (e.g., static rules with priority 1) are executed in an arbitrary order which is not defined and can change between runs or browser versions. This is the case across browsers.
+Avoid writing rules with the same action and priority that must run in a particular order. Rules with the same action and priority (e.g., redirect rules with priority 1) are executed in an arbitrary order which is not defined and can change between runs or browser versions, even when spread between multiple types of ruleset (e.g, a static rule and a session rule). This is the case across browsers.
```

Comment:
>(e.g, a static rule and a session rule)

The rest of this looks good, but it's not clear to me what the last part is adding. Tell *me* what it means in as many words as it takes. We'll loop back to the wording of this afterwards.

Reply:
For sure! If you have two redirect rules with the same priority, with one added in the manifest and the other as a dynamic rule, then there are no guarantees about where you'll be redirected to. We might prefer session rules over dynamic rules in this Chrome version but in the next prefer dynamic rules first and then session rules second.